deps/build.jl

```diff
@@ -21,4 +21,10 @@ end
     provides(Homebrew.HB, "hdf5", hdf5, os = :Darwin )
 end
 
+begin
+    hdf5 = library_dependency("libhdf5")
```

Comment:
I think this needs to be refactored a bit to avoid overwriting the old value of `hdf5` - make the aliases os-dependent (or maybe don't bother and just add all possible aliases everywhere?) then put the os-dependent providers last?